repository/Seaside-WebSocket-Core.package/JSWebSocket.class/instance/javascriptContentOn..st

```diff
@@ -0,0 +1,8 @@
+printing
+javascriptContentOn: aStream
+	"TODO: window.location should not be retrieved here..."
```

Comment:
Should we use `#baseUrl` or something similar here instead?

Reply:
Indeed. It should probably use the settings of WARequestHandlingConfiguration... it's just that I do not prefer that way of working. Alas, I see no alternative either.